docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,205 @@
+
+User can set this property in properties.xml in the following way
+
+```xml
+<property>
+    <name>dt.application.{ApplicationName}.operator.{OperatorName}.prop.jsonSchema</name>
+    <value>{
+      "padding": "_",
+      "alignment": "left",
+      "fields": [
+      {
+      "name": "adId",
+      "type": "Integer",
+      "length": "3",
+      "padding": "0"
+      },
+      {
+      "name": "campaignId",
+      "type": "Integer",
+      "length": "3",
+      "padding": " "
+
+      },
+      {
+      "name": "adName",
+      "type": "String",
+      "length": "10",
+      "alignment":"right"
+      },
+      {
+      "name": "bidPrice",
+      "type": "Double",
+      "length": "3"
+      },
+      {
+      "name": "startDate",
+      "type": "Date",
+      "format": "yyyy-MM-dd HH:mm:ss",
+      "length": "19"
+
+      },
+      {
+      "name": "endDate",
+      "type": "Date",
+      "format": "dd/MM/yyyy",
+      "length": "10"
+      },
+      {
+      "name": "securityCode",
+      "type": "Long",
+      "length": "5"
+      },
+      {
+      "name": "active",
+      "type": "Boolean",
+      "length": "4",
+      "trueValue": "true",
+      "falseValue": "false"
+      },
+      {
+      "name": "optimized",
+      "type": "Boolean",
+      "length": "1",
+      "trueValue": "y",
+      "falseValue": "n"
+      },
+      {
+      "name": "parentCampaign",
+      "type": "String",
+      "length": "10"
+      },
+      {
+      "name": "weatherTargeted",
+      "type": "Character",
+      "length": "1"
+      },
+      {
+      "name": "valid",
+      "type": "String",
+      "length": "10",
+      "padding":" "
+      }
+      ]
+      }
+    </value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application.
+
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+
+User can set this property in properties.xml in the following way
+
+```xml
+<property>
+    <name>dt.application.{ApplicationName}.operator.{OperatorName}.port.out.attr.TUPLE_CLASS</name>
+    <value>fixedwidthparser.Ad</value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application and "fixedwidthparser.Ad" is the Tuple class.
+
+Following code can be added to `populateDAG()` method of application to set Tuple Class:
+```java
+dag.setOutputPortAttribute({OperatorName}.out, Context.PortContext.TUPLE_CLASS, fixedwidthparser.Ad)
```

Comment:
Please confirm that you've verified this code fragment in an actual working application.